src/test/java/qna/domain/UserRepositoryTest.java

```diff
@@ -25,8 +34,7 @@ void findByUserId() {
         User user = new User("jurl", "1234", "주디", "dbswnfl2@");
         userRepository.save(user);
 
-        userRepository.findByUserId(user.getUserId()).ifPresent(
-                it -> assertThat(it).isEqualTo(user)
-        );
+        Optional<User> byUserId = userRepository.findByUserId(user.getUserId());
```

Comment:
보통 실제 테스트 결과의 변수명은 actual을 붙입니다.

```suggestion
        Optional<User> actual = userRepository.findByUserId(user.getUserId());
```